Source/WebDriver/Session.h

```diff
@@ -159,7 +159,8 @@ public RefCounted<Session>
 #if ENABLE(WEBDRIVER_BIDI)
     void enableGlobalEvent(const String&);
     void disableGlobalEvent(const String&);
-    void dispatchEvent(RefPtr<JSON::Object>&&);
+    void dispatchBiDiMessage(RefPtr<JSON::Object>&&);
+    void relayBidiCommand(const String&, unsigned commandId, Function<void(WebSocketMessageHandler::Message&&)>&&);
```

Comment:
Might be possible to use `CompletionHandler` instead of `Function` here. You'd need to clear out `m_relayedCommands` in the destructor to avoid asserting.

Reply:
I went with `Function<...>` because, IIUC, the spec does not require replying to pending commands when the session ends, either through `session.end()`[1] or `browser.close()`[2]. In both cases, the client is notified by the closure of the WebSocket connection.

[1] https://w3c.github.io/webdriver-bidi/#commands-sessionend
[2] https://w3c.github.io/webdriver-bidi/#command-browser-close